source/module_esolver/esolver_sdft_pw_tool.cpp

```diff
@@ -104,7 +104,7 @@ void ESolver_SDFT_PW::sKG(const int nche_KG, const double fwhmin, const double w
     double dw =  dw_in / ModuleBase::Ry_to_eV; //converge unit in eV to Ry 
     double sigma = fwhmin / TWOSQRT2LN2 / ModuleBase::Ry_to_eV;
     double dt = dt_in; //unit in a.u., 1 a.u. = 4.837771834548454e-17 s
-    int nt = ceil(sqrt(20)/sigma/dt);
+    int nt = ceil(sqrt(36.84)/sigma/dt); //exp(-18.42) = 1e-8
```

Comment:
There are so many hard code , can you define these parameters in this cpp file using 
1. "#define XX 36.84"
2. "const double XX 36.84"